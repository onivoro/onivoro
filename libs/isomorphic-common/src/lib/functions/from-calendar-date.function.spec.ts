import { fromCalendarDate } from './from-calendar-date.function';

describe('fromCalendarDate', () => {
    describe('GIVEN input is partial or full ISO8601', () => {
        it.each([
            '2023-12-23',
        ])('%j', (input) => {
            expect(fromCalendarDate(input)).toMatchSnapshot();
        });
    });
});