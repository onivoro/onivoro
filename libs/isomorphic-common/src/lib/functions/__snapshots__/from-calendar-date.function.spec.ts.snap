// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fromCalendarDate GIVEN input is partial or full ISO8601 "2023-12-23" 1`] = `2023-12-22T12:00:00.000Z`;
